# Vehicle Center Kinematics on Vehicle Coordinates, MKS Units Mandatory.
float64[2]   CM_Velocity
float64[2]   CM_Acceleration
float64     Vehicle_Heading
float64     CM_AngularVel
float64     CM_AngularAcc

# Wheel Kinematics
float64[4]   Wheel_LinearVel
float64[4]   Wheel_SteerAngl

# Time Stamp
time        Time_Stamp
